arch/arm64/boot/dts/qcom/sdm632-motorola-ocean.dts

```diff
@@ -89,14 +89,60 @@
 
 &msmgpio {
 	gpio-reserved-ranges = <96 4>;
+
+	pmx_mdss_default: pmx_mdss_default {
```

Comment:
indentation is funky
Use hardware tabs whenever possible

Reply:
Gotcha. Fixed.